roles/kustomize_deploy/tasks/01_install_operators.yml

```diff
@@ -0,0 +1,206 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: OLM resources
+  block:
+    - name: Perform kustomization on the OLM directory
```

Comment:
ditto. This can be reduced to one single task, the next one, with the kustomization as a variable. Quicker and cheaper.

Reply:
ack!